build.gradle

```diff
@@ -1,5 +1,10 @@
-plugins {
-    id 'com.android.application' version '7.4.2' apply false
-    id 'com.android.library' version '7.4.2' apply false
+buildscript {
+    dependencies {
+        classpath 'org.jacoco:org.jacoco.core:0.8.9'
+    }
 }
 
+plugins {
+    id 'com.android.application' version '8.0.0' apply false
+    id 'com.android.library' version '8.0.0' apply false
```

Comment:
Please don't try to sneak controversial changes into develop. #89 is still contested. >:(

Reply:
I think for now i will revert back to 7.4.2 so we can come back to the topic of upgrading at a later point in time. This should resolve the issue.